frontend/src/router/router-typed.d.ts

```
@@ -0,0 +1,13 @@
+import type { WelcomeRouteInfo } from '@/modules/welcome';
+import type { SchemasRouteInfo } from '@/modules/schemas';
+
+type RouteNamedMap = WelcomeRouteInfo & HomeRouteInfo & SchemasRouteInfo;
+
+declare module 'vue-router' {
+  interface TypesConfig {
+    RouteNamedMap: RouteNamedMap;
+  }
+}
+
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)